examples/CMakeLists.txt

```diff
@@ -30,15 +30,17 @@ foreach(example_src IN LISTS example_srcs)
 endforeach()
 
 # Silence some warnings from old thrust headers:
-set(thrust_examples
+set(nvbench_examples
```

Comment:
This was intentionally named `thrust_examples`. Older versions of thrust emitted warnings from headers, and these are the examples that use thrust and needed the suppressions.